daphne/src/lib.rs

```diff
@@ -225,6 +236,25 @@ pub struct ProblemDetails {
     pub detail: Option<String>,
 }
 
+/// Global DAP parameters common across tasks.
+//
+// TODO spec: `max_window_size`, `min_duration_start`, and `max_duration_end`
+//            should be reflected to the DAP spec.
+#[derive(Deserialize, Serialize)]
+pub struct DapGlobConfig {
+    /// Maximum window size permitted in CollectReq.
+    /// Prevents Collectors from requesting wide range or reports.
+    pub max_batch_duration: u64,
+
+    /// Lower bound of an acceptable batch window in CollectReq.
+    /// Batch intervals must start after `min_batch_interval_start`.
+    pub min_batch_interval_start: u64,
```

Comment:
Is this a fixed point in time, or a duration between the current time and the min batch interval start? I think it should be the latter, otherwise we'd have to update this periodically.

Reply:
`min_batch_interval_start` and `max_batch_interval_end` are duration between the current time and the min/max batch interval start/end.
Do you have a better name for this variable?

Reply:
These names are fine.